bin/clean-tsv-metadata

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+import pandas as pd
+import argparse
+
+def clean_metadata_file(file_name, n_cols, header):
+    """
+    1. Read the metadata TSV file
+    2. Sort alphabetically and make sure n columns (n-1 tabs) each row
+    3. Write out (overwrite) metadata file
+    """
+    col_names = list(pd.read_csv(file_name, sep="\t", header=None, nrows=1).iloc[0]) if header else list(range(n_cols))
```

Comment:
It's unclear to me what value reading in the column names (header) separately adds to this script. 

If I run `pd.read_csv()` with `header=None`, then the columns are automatically named using zero-indexed integers. 

Additionally, I can think of an edge case where we may want to specify `--header` and also enforce a number of columns, which the code as written does not currently support.

I would recommend instead dropping the `col_names` assignment line, reading in the TSV with the `header` argument passed, and handling surplus columns in a separate part of the code.

Reply:
See f95f2de, I couldn't think of a smarter way than reading it in a separate time.